src/content/api/merchant-configs.mdx

```diff
@@ -0,0 +1,376 @@
+---
+title: Merchant Configs
+description: Merchant Config model and related endpoints
+draft: true
+nav:
+  path: API
+  order: 26
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Error from '../../components/Error.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+import CodePanel from '../../components/CodePanel.astro';
+
+A Merchant Config defines the available payment options for paying a [Payment Request](https://docs.centrapay.com/api/payment-requests).
+
+## Merchant Config Model
+
+### Attributes
+
+<Properties>
+  <Property name="paymentOptions" type="array">
+    Array of [Payment Option Configs](#payment-option-config-model).
+  </Property>
+
+  <Property name="createdAt" type="timestamp">
+    When the Merchant Config was created.
+  </Property>
+
+  <Property name="createdBy" type="crn">
+    The User or API Key that created the Merchant Config.
+  </Property>
+
+  <Property name="updatedAt" type="timestamp">
+    When the Merchant Config was updated.
+  </Property>
+
+  <Property name="updatedBy" type="crn">
+    The User or API Key that updated the Merchant Config.
+  </Property>
+
+  <Property name="allowedRedirectUrls" type="array" experimental>
+    Allowed prefixes for the `redirectUrl` property on Payment Requests created with this Config.
+  </Property>
+</Properties>
+
+---
+
+## Payment Option Config Model
+
+### Attributes
+<Properties>
+  <Property name="type" type="string" required>
```

Comment:
```suggestion
  <Property name="type" type="string">
```

We haven't been adding required attributes on models?

Reply:
Yeah I thought this case was slightly different as there is an endpoint that requires an array of this model so it needs to be clear which of the fields are required